src/pages/workspace/expensifyCard/WorkspaceCardsListLabel.tsx

```diff
@@ -57,7 +57,10 @@ function WorkspaceCardsListLabel({type, value, style}: WorkspaceCardsListLabelPr
     const [cardSettings] = useOnyx(`${ONYXKEYS.COLLECTION.PRIVATE_EXPENSIFY_CARD_SETTINGS}${workspaceAccountID}`);
     const paymentBankAccountID = cardSettings?.paymentBankAccountID;
 
-    const isConnectedWithPlaid = useMemo(() => !!bankAccountList?.[paymentBankAccountID ?? 0]?.accountData?.additionalData?.plaidAccountID, [bankAccountList, paymentBankAccountID]);
+    const isConnectedWithPlaid = useMemo(() => {
```

Comment:
Can we add a comment here explaining why we fallback on `bankAccountData?.additionalData?.plaidAccountID` and also include a link to the clean up issue https://github.com/Expensify/App/issues/47944?

Reply:
@MariaHCD Updated in https://github.com/Expensify/App/pull/48984/commits/1a2c7429197885f8160385197f88ac4d47b74331. Let me know what do you think